Marlin/configuration_store.cpp

```diff
@@ -1088,7 +1088,7 @@ void MarlinSettings::postprocess() {
           stepperE4.setCurrent(val, R_SENSE, HOLD_MULTIPLIER);
         #endif
       #else
-        for (uint8_t q = 11; --q;) EEPROM_READ(val);
+        for (uint8_t q = 0; q < 11; q++) EEPROM_READ(val);
```

Comment:
This doesn't change the read order, ya know.

Reply:
Yes...   And that is why I reverted #8284.   I didn't test the bugfix-v2.0.0 branch, but it looked like it was OK.

Reply:
Oh duh... I'm smart...